### PR TITLE
Updated README.md with information on dependencies

### PR DESCRIPTION
**Pull request by damontgomery**
Added information on adding custom scripts using `drupal-ti --include [script]` and provided an example that loads a module's dependencies.

Note: This information was scattered within the project issue queue, PRs, and codebase. I feel like this is a common use case that deserves an example in the readme.

### CODE REVIEW

**Comment by davereid** [comment]
:+1: for this, and I also think adding a line for modules that use Composer Manager could also use an example. In url_embed we use the following: https://github.com/drupal-media/url_embed/blob/8.x-1.x/.travis-before-script.sh

File: README.md [hunk 1]
```diff
@@ -86,6 +86,51 @@ Example:
...
+# Ensure the right Drupal version is installed.
+# The first time this is run, it will install Drupal.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
+
+# Change to the Drupal directory
+cd "$DRUPAL_TI_DRUPAL_DIR"
+
+# Manually clone the dependencies
+git clone https://github.com/my-project/my-repo.git modules/my_module
```
Comment (davereid) [inline README.md]: Note the example here only works for Drupal 8. To make it work across all versions, I would recommend using the following example:
```
# Download dependencies
mkdir -p "$DRUPAL_TI_DRUPAL_DIR/$DRUPAL_TI_MODULES_PATH"
cd "$DRUPAL_TI_DRUPAL_DIR/$DRUPAL_TI_MODULES_PATH"
git clone --depth 1 --branch 8.x-1.x http://git.drupal.org/project/my_dependency.git
```

Comment (damontgomery) [inline README.md]: Thanks @davereid. I'll put together an update hopefully tomorrow. I'm not very familiar with using composer manager, but I'll use your link as a reference.

**Comment by damontgomery** [comment]
@davereid, I updated this PR with your recommendations. Thanks for the examples. I think it makes the readme much longer, but I feel like these are two very common use cases for this module, especially for D8 where Composer Manager is becoming more popular.

**Comment by LionsAd** [comment]
Thanks this is great!

**Pull request merged**